kotlin-test/src/test/kotlin/org/axonframework/extension/kotlin/test/FixtureExtensionsTest.kt

```diff
@@ -1,15 +1,52 @@
 package org.axonframework.extension.kotlin.test
 
-import org.axonframework.test.aggregate.AggregateTestFixture
 import kotlin.test.Test
 
 internal class FixtureExtensionsTest {
 
+    @Test
+    fun `Aggregate test fixture extension should create an aggregate fixture`() {
+        aggregateTestFixture<ExampleAggregate>()
+    }
+
+    @Test
+    fun `Saga test fixture extension should create a saga fixture`() {
+        sagaTestFixture<ExampleSaga>()
+    }
+
+    @Test
+    fun `Whenever extension should apply to an aggregate fixture`() {
+        val fixture = aggregateTestFixture<ExampleAggregate>()
+
+        fixture
+                // Call on an AggregateTestFixture instance
+                .whenever(ExampleCommand("id"))
+
+        fixture
+                // Call on an AggregateTestFixture instance
+                .whenever(ExampleCommand("id"), mapOf())
+    }
+
+    @Test
+    fun `Whenever extension should apply to a result validator`() {
+        val fixture = aggregateTestFixture<ExampleAggregate>()
+
+        fixture
+                .given()
+                // Call on a ResultValidator instance
+                .whenever(ExampleCommand("id"))
+
+        fixture
+                .given()
```

Comment:
I'd use `givenNoPriorActivity()`. It's the same, true, but it's a little clearer I think.